src/Microsoft.ML.OnnxTransform/DnnImageFeaturizerTransform.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// This is a helper class that is required to use the <see cref="DnnImageFeaturizerEstimator"/>.
+    /// Note that by default, it is not usable as it does not have any valid methods that return an <see cref="EstimatorChain{TLastTransformer}"/>
+    /// that is used by the DnnImageFeaturizeEstimator.
+    /// In order to use this, at least one model project with the corresponding extension methods must by included.
+    /// See Microsoft.ML.DNNImageFeaturizer.ResNet18 for an example.
+    /// </summary>
+    public sealed class DnnImageModelSelector
+    {
+    }
+
+    /// <summary>
+    /// This is a helper class used to store all the inputs to an extension method on a DnnImageModelSelector required to return
+    /// a chain of two <see cref="OnnxScoringEstimator"/>s.
+    /// </summary>
+    public sealed class DnnImageFeaturizerInput
+    {
+        public readonly IHostEnvironment Env;
```

Comment:
These should be properties, and not fields.

`public IHostEnvironment Env { get; }`